pyproject.toml

```diff
@@ -7,7 +7,7 @@ requires = [
 build-backend = "setuptools.build_meta"
 
 [project]
-name = "coco-postprocess"
+name = "cocopp"
```

Comment:
To publish on pypi. the package name needs to be the same

Reply:
👍